### ISSUE TITLE
Running dotnet run on a console app has no output

### ISSUE THREAD

**Issue by zigpot** [dotnet/sdk#23616]
I followed this tutorial https://dotnet.microsoft.com/en-us/learn/dotnet/hello-world-tutorial/run
Operating System: Debian GNU/Linux 11 (bullseye) amd64

![image](https://user-images.githubusercontent.com/16068753/151103757-45604346-93de-4796-8a06-ca7ccb66f814.png)

As can be seen, running:

> dotnet run

inside the project didn't output anything.

here is _Program.cs_:
```
﻿using System;

namespace MyApp
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Hello World!");
        }
    }
}
```

and _MyApp.csproj_:
```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net5.0</TargetFramework>
  </PropertyGroup>

</Project>
```

Surely I can do

> dotnet ./bin/Debug/net5.0/MyApp.dll

and it would run, but that isn't how it was in the tutorial

How can I run this simple program, specifically using 

> dotnet run

alone?

**Comment by marcpopMSFT** [comment]
@zigpot Does this repro with the 6.0.100 SDK and targeting net6.0? I can't reproduce this behavior on windows with either the 6.0 or 5.0 SDK so I'm not sure if it's specific to debian.

**Comment by zigpot** [comment]
@marcpopMSFT I'm sorry, I haven't opened Github in a while.

This behaviour was not specific to Debian as my previous attempt in (if I recalled correctly) Ubuntu 20.04, yielded the same result. And just like you, I was neither able to reproduce it on Windows.

I have avoided using version 6 due to [another issue](https://github.com/dotnet/core/issues/4446) I was also experiencing

I have snap installed latest/beta which happens to be dotnet SDK version 7.0.0, and the all the problems disappear.

**Comment by marcpopMSFT** [comment]
Thanks for the detailed reply. Would you be able to try the recently released 6.0.200 version? It sounds like it affects linux but not windows and affects 6.0.102 but not 7.0.100. I'm just trying to narrow down where the issue might be.

**Comment by ckyleda** [comment]
Why was this issue closed? It still exists. I can launch the project via the dll, but dotnet run simply terminates with no console output (aside from "using launchsettings.json"). This is for an ASPNET Core webapp. Using NET 3.1 on Ubuntu installed via Snap

Was there any solution discovered to this?
